### DISCUSSION TITLE
Non-Subscription Plan

### DISCUSSION THREAD

**Discussion by rangav** [thunderclient/thunder-client-support#1547] [Ideas]
## Non-Subscription Plan
We may launch a non-subscription plan for **individuals** with a subset of features available in subscription plans.

- If you are interested in this plan, please click the **upvote** to express your interest. 
- If more than **250 users** like this plan, we will consider launching it.

### Questions
- If you have any questions, please contact us using [this form](https://www.thunderclient.com/contact).

--------
<img width="295" alt="Screenshot 2024-05-24 at 14 12 19" src="https://github.com/thunderclient/thunder-client-support/assets/8637550/422e4e2b-8cf9-4193-b535-d9cc44af623a">

**Comment by pdougla540** [comment]
@rangav when could we expect this to be a one-time purchase option? Thanks!

**Reply by rangav** [reply]
We will plan for implemention after more than 250 users upvote the above item.
